content/en/blog/2023-04-01-march-update/index.md

```diff
@@ -0,0 +1,256 @@
+with new features and improvements. Users are encouraged to upgrade for the best experience.
+
+Features and improvements
+
+- Experimental support of drift detection of Helm releases compared to cluster-state.
+- Improved handling of `SIGTERM` signals received by the helm-controller, which will now
+  terminate running Helm install or upgrade actions, instead of potentially leaving them
+  in a pending state.
+- Opt-in OOM watcher in helm-controller to handle graceful termination of the controller
+  before it is forcefully killed by Kubernetes' OOM killer.
+- Kubernetes client and Custom Resource Definition life-cycle improvements to reduce the
+  memory consumption of the helm-controller, with observed reductions up to 50%.
+- Opt-in allowance of DNS lookups during the rendering of Helm templates in the
+  helm-controller via feature gate.
+- Optional disabling of the cache of the status poller used to determine the health of
+  the resources applied by the kustomize-controller. This may improve memory usage on
+  large scale clusters at the cost of more direct API calls.
+- Changes to the logging of all controllers to ensure Kubernetes components like the
+  discovery client use the configured logging format.
+- New `flux events` command to display Kubernetes events for Flux resources, including
+  the events of a referenced resource.
+- Custom annotations can now be set with `flux push` using `--annotations`.
+
+New documentation
+
+- Cheatsheet: [Enable Helm drift
+  detection](/flux/cheatsheets/bootstrap/#enable-helm-drift-detection)
+- Cheatsheet: [Enable Helm near OOM
+  detection](/flux/cheatsheets/bootstrap/#enable-helm-near-oom-detection)
+- Cheatsheet: [Allow Helm DNS
+  lookups](/flux/cheatsheets/bootstrap/#allow-helm-dns-lookups)
+- Controller: [New helm-controller feature gates and
+  options](/flux/components/helm/options/#feature-gates)
+- Controller: [New kustomize-controller feature
+  gate](/flux/components/kustomize/options/#feature-gates)
+- Spec: [HelmRelease drift
+  detection](/flux/components/helm/helmreleases/#drift-detection)
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.19.0>
+
+#### Terraform-controller
+
+#### Flux Subsystem for Argo
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```

Comment:
@juozasg  Anything you'd like to say here for March?

Reply:
Version 0.24.0 of the VS Code extension was released. This version introduces a new feature for users of Weave GitOps Enterprise - Templates. Using Templates users can create complex GitOps configurations, workflows and pipelines with a simple UI. Templates must be enabled in the VS Code settings to be available. The README.md includes additional information.

We are also continuing to work on UI and performance improvements. With 0.24 release Sources and Workloads are now grouped by Namespaces and the information is presented in a more consistent way.